backend/algorithms/objects/conditions.py

```diff
@@ -197,18 +210,20 @@ def validate(self, user: User) -> tuple[bool, list[str]]:
         """
         Determines if the user has met the WAM condition for this category.
 
-        Will always return True and a warning since WAM can fluctuate
+        Will always return False and a warning since WAM can fluctuate
         """
         warning = self.get_warning(user.wam(self.category))
-        return True, [warning] if warning else []
+        return True, [warning]
 
-    def get_warning(self, applicable_wam: Optional[float]) -> str | None:
+    def get_warning(self, applicable_wam: Optional[float]) -> str:
         """ Returns an appropriate warning message or None if not needed """
+        category = re.sub(r"courses && ([A-Z]{4}) courses", r"\1 courses", str(self.category))
+        wam_warning = f"Requires {self.wam} WAM in {category}. "
         if applicable_wam is None:
-            return f"Requires {self.wam} WAM in {self.category}. Your WAM in {self.category} has not been recorded"
+            return wam_warning + f"Your WAM in {category} has not been recorded"
```

Comment:
since wam_warning is that string anyways, could just make this a f-string perhaps?